lambda_functions/v1/functions/flask_app/app/api/sirius_service.py

```diff
@@ -269,7 +269,9 @@ def format_sirius_response(sirius_response=None, sirius_response_code=500):
                     "type": sirius_response["type"],
                     "id": sirius_response["uuid"],
                     "attributes": {
-                        "submission_id": sirius_response["metadata"]["submission_id"],
```

Comment:
This method has been deprecated since you made this change, however it's an easy fix. Just make the same change to the new method `format_sirius_success` instead